app/models/iterations/project_iterations.rb

```diff
@@ -53,21 +53,25 @@ def iteration_length_in_days
       project.iteration_length * DAYS_IN_A_WEEK
     end
 
-    def project_start_date
-      project.start_date
+    def project_start_from_first_sprint
+      project.start_date.wday - project.iteration_start_day
     end
 
-    def start_date(iteration_number)
+    def first_sprint_start_date
```

Comment:
What do you think about renaming this method in a way to show that this date is the one the first sprint _should_ have started? Also, it would be easier to understand if we add a `.days` in the end of the second operand, like this:

```ruby
project.start_date - project_start_from_first_sprint.days
```